### PR TITLE
Feat/rebranding

### PR DESCRIPTION
**Pull request by isaackps**
## Summary

Update website to the new rebranding logo and styles

## Changes

- Update new logo
- Change color schemes
- Fix some css issues

## Issues

https://www.pivotaltracker.com/n/projects/2424361/stories/182152202

### CODE REVIEW

File: src/components/FooterBar/FooterBar.tsx [hunk 1]
```diff
@@ -43,14 +43,21 @@ const getData = (configFile?: ConfigFile) => {
...
         { label: "Create Documents", to: "/", render: renderNavLink },
       ],
     },
     {
       category: "Resources",
       items: [
-        { label: "Webinars", to: `${networkPath}/learn`, render: renderExternalLinkWithoutIcon },
+        { label: "Learn", to: `${networkPath}/learn`, render: renderExternalLinkWithoutIcon },
+        { label: "FAQ", to: `${networkPath}/faq`, render: renderExternalLinkWithoutIcon },
+        { label: "ETA", to: `${networkPath}/eta`, render: renderExternalLinkWithoutIcon },
```
Comment (simboonlong) [inline src/components/FooterBar/FooterBar.tsx:55]: question (non-blocking): there are only a few internal links. should we change external links from `renderExternalLinkWithoutIcon` to  `renderExternalLink`? (so as to add the icon)

perhaps to discuss if it's still make sense to make creator "look" like part of tradetrust website via similar header, footer (we know that they are separate sites) <- there wasn't any conclusion iirc

File: src/components/FormSelectionContainer/FormSelect/FormSelect.tsx [hunk 2]
```diff
@@ -134,15 +134,15 @@ export const FormSelect: FunctionComponent<FormSelectProps> = ({ id, form, onAdd
...
+              : "text-cerulean-500 hover:text-cerulean-800 hover:bg-cloud-100"
           }`}
           onClick={() => handleForm()}
           {...props}
         >
           {formStatus === FormStatus.PENDING ? (
             <div className="flex flex-col flex-wrap">
               <div>{form.name}</div>
-              <LoaderSpinner className="content-center self-center mt-1" />
+              <LoaderSpinner className="content-center self-center mt-1" primary="#2D5FAA" />
```
Comment (simboonlong) [inline src/components/FormSelectionContainer/FormSelect/FormSelect.tsx:145]: `primary` perhaps to put in `constants` folder and export from there

File: src/components/ProcessDocumentScreen/ProcessDocumentTitle/ProcessDocumentTitle.tsx [hunk 3]
```diff
@@ -26,7 +26,7 @@ export const ProcessDocumentTitle: FunctionComponent<ProcessDocumentTitle> = ({
       case QueueState.PENDING:
         return (
           <>
-            <LoaderSpinner className="mr-2" width="24px" primary="#3B8CC5" />
+            <LoaderSpinner className="mr-2" width="24px" primary="#2D5FAA" />
```
Comment (simboonlong) [inline src/components/ProcessDocumentScreen/ProcessDocumentTitle/ProcessDocumentTitle.tsx:29]: `primary` perhaps to put in `constants` folder and export from there

File: src/components/ProcessDocumentScreen/ProcessedDocumentTag/ProcessedDocumentTag.tsx [hunk 4]
```diff
@@ -45,35 +45,35 @@ export const ProcessedDocumentTag: FunctionComponent<PublishedTagProps> = ({
             className="mr-4 flex-shrink-0"
             data-testid="processing-loader"
             width="48px"
-            primary="#3B8CC5"
+            primary="#2D5FAA"
           />
```
Comment (simboonlong) [inline src/components/ProcessDocumentScreen/ProcessedDocumentTag/ProcessedDocumentTag.tsx:49]: `primary` perhaps to put in `constants` folder and export from there

File: src/components/RevokeContainer/RevokeDocumentDropZone/RevokeDocumentDropZone.tsx [hunk 5]
```diff
@@ -88,19 +88,19 @@ export const RevokeDocumentDropZone: FunctionComponent<RevokeDocumentDropZone> =
...
             )}
             {documentUploadState !== DocumentUploadState.LOADING && (
-              <Button className="bg-cerulean text-white hover:bg-cerulean-500 mt-4">Select Document</Button>
+              <Button className="bg-cerulean-500 text-white hover:bg-cerulean-800 mt-4">Select Document</Button>
             )}
             {documentUploadState === DocumentUploadState.LOADING && (
               <div className="py-8 h-72 flex flex-col items-center justify-center" data-testid="dropzone-loader">
-                <LoaderSpinner primary="#3B8CC5" width="40px" />
-                <h4 className="mt-8 text-cerulean">Verifying Document</h4>
+                <LoaderSpinner primary="#2D5FAA" width="40px" />
```
Comment (simboonlong) [inline src/components/RevokeContainer/RevokeDocumentDropZone/RevokeDocumentDropZone.tsx:102]: `primary` perhaps to put in `constants` folder and export from there

File: src/components/FooterBar/FooterBar.tsx (hunk 1 above)
Comment (isaackps) [reply src/components/FooterBar/FooterBar.tsx:55]: ok can i was thinking of changing to externalLink too.

but whats the issue with creator to "look" like part of tradetrust website? isn't it suppose to be like that?

File: src/components/FormSelectionContainer/FormSelect/FormSelect.tsx (hunk 2 above)
Comment (isaackps) [reply src/components/FormSelectionContainer/FormSelect/FormSelect.tsx:145]: ok

File: src/components/ProcessDocumentScreen/ProcessDocumentTitle/ProcessDocumentTitle.tsx (hunk 3 above)
Comment (isaackps) [reply src/components/ProcessDocumentScreen/ProcessDocumentTitle/ProcessDocumentTitle.tsx:29]: ok

File: src/components/ProcessDocumentScreen/ProcessedDocumentTag/ProcessedDocumentTag.tsx (hunk 4 above)
Comment (isaackps) [reply src/components/ProcessDocumentScreen/ProcessedDocumentTag/ProcessedDocumentTag.tsx:49]: ok

File: src/components/RevokeContainer/RevokeDocumentDropZone/RevokeDocumentDropZone.tsx (hunk 5 above)
Comment (isaackps) [reply src/components/RevokeContainer/RevokeDocumentDropZone/RevokeDocumentDropZone.tsx:102]: ok

File: src/components/FooterBar/FooterBar.tsx (hunk 1 above)
Comment (simboonlong) [reply src/components/FooterBar/FooterBar.tsx]: i think there was once mentioned to drop the header navbar and footer links, reverting to the previous stance of a standalone application, due to the confusion of leading off to tradetrust website (ux concern)

we can leave it as it is first, yeah ok with `externalLink`

**Review by simboonlong — APPROVED** [review]

**Pull request merged**